Resources/Prototypes/Entities/Objects/Misc/utensils.yml

```
@@ -53,6 +54,7 @@
     types:
     - Fork
   - type: MeleeWeapon
+    animationRotation: 180 # WWDP
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)